cypher/cypher-docs/src/docs/dev/syntax/comments.asciidoc

```diff
@@ -1,7 +1,16 @@
 [[cypher-comments]]
 = Comments
 
-To add comments to your queries, use double slash. Examples:
+[abstract]
+--
+Cypher supports comments in a query.
+A comment begin with double slash (`//`) and continue to the end of the line.
+Comments do not execute, they are for humans to read.
+--
+
+To add comments to your queries, use double slash (`//`).
```

Comment:
```suggestion
A comment begins with double slash (`//`) and continue to the end of the line.
Comments do not execute, they are for humans to read.
```
Unsure though if we need the last line at all. I do believe users are familiar with how comments work.